app/src/pages/HomePage.tsx

```diff
@@ -4,10 +4,14 @@ import { Alignment, Box, Button, ContainingView, Direction, Image, KibaIcon, Lay
 
 import { useAccount, useOnLinkAccountsClicked } from '../AccountContext';
 import { ImageView } from '../components/ImageView';
+import { UpdateResult, UploadImage } from '../components/UploadFile';
+import { useGlobals } from '../globalsContext';
 import imageData from '../imageData.json';
 
 export const HomePage = (): React.ReactElement => {
   const account = useAccount();
+  const { web3StorageClient } = useGlobals();
+
   const [stage, setStage] = React.useState<number>(1);
   const [imageLink, setImageLink] = React.useState<string>('');
```

Comment:
this should be null to start with (you should bascially never use an empty string)

Reply:
also imageUrl, not imageLink